cms/templates/500.html

```diff
@@ -8,20 +16,24 @@
   <section class="content">
     <header>
       <h1 class="title title-1">
-        ${_("The {studio_name} servers encountered an error").format(
-          studio_name=u"<em>{studio_name}</em>".format(studio_name=settings.STUDIO_SHORT_NAME)
+        ${_(u"The {em_start}{studio_name}{em_end} servers encountered an error").format(
```

Comment:
I think this one actually does need to be wrapped in `Text()` -- third example in [html-context-in-mako](http://draft-safe-templates-2016-03-23.readthedocs.org/en/latest/conventions/safe_templates.html#html-context-in-mako).